test/android/util/runtime_client/src/org/xwalk/test/util/XWalkRuntimeClientTestGeneric.java

```diff
@@ -17,12 +20,20 @@
         extends ActivityInstrumentationTestCase2<T> {
     private XWalkRuntimeView mRuntimeView;
     XWalkRuntimeClientTestUtilBase mTestUtil;
+    private Timer mTimer = new Timer();
 
     @Override
     protected void setUp() throws Exception {
         super.setUp();
+        waitForXWalkReady();
```

Comment:
They run in the same thread, test case sometimes failed since the initialization was not completed in the super class.